packages/mobile/src/store/common/reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonState } from 'audius-client/src/common/store'
+
+import { CommonActions, RECEIVE } from './actions'
+
+// Not using reduxjs/toolkit here because immer performance was less than optimal
+// for this reducer that gets hit very often
+export const reducer = (
```

Comment:
from what ive seen, its still standard to do default export for reducer, but def a nit

Reply:
I say we do named exports across the board